src/main/java/adris/altoclef/tasks/ResourceTask.java

```diff
@@ -247,6 +260,23 @@ public ResourceTask forceDimension(Dimension dimension) {
 
     protected abstract String toDebugStringName();
 
+    @Override
+    public boolean shouldEmptyCursorSlot(AltoClef mod, Task candidate) {
+        return false;
+    }
+
+    @Override
```

Comment:
This method ensures that the 2x2 crafting grid is always clear so items cannot get stuck. The only time this doesn't happen is if we are crafting in our inventory.